lua/packer.lua

```diff
@@ -320,9 +322,8 @@ packer.update = function(...)
     local start_time = vim.fn.reltime()
     local results = {}
     await(clean(plugins, results))
-    local missing_plugins, installed_plugins = util.partition(
-                                                 plugin_utils.find_missing_plugins(plugins),
-                                                 update_plugins)
+    local missing = await(plugin_utils.find_missing_plugins(plugins))
+    local missing_plugins, installed_plugins = util.partition(missing, update_plugins)
```

Comment:
Does this need the `missing or {}` check included in the `sync` case?

Reply:
Probably 😅

Reply:
Actually looking at the functions implementation it always returns a table so I don't think any of the `or {}` are required.